egs/mucs21_subtask2/asr1/local/prepare_data.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+#local/prepare_data.sh data/hi-en/test/transcripts/wav.scp data/hi-en/test/audio/ out.scp
```

Comment:
ditto
Could you briefly explain what this script does?

Reply:
This script replicates gen_wavscp functionality(I can remove gen_wavscp). It is necessary to change the paths present in the pipeline Shreya has for 2 datasets.